src/index.js

```diff
@@ -1,17 +1,22 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+// IMPORT SCSS FILE TO BE SOURCE OF STYLING
+import "./styles.scss";
```

Comment:
It looks like here "styles.scss" is being imported but the scss file is just called "style.scss". I think we should probably use "styles.scss".